src/admin/mod.rs

```diff
@@ -44,9 +44,9 @@ pub struct Server {
 pub struct Readiness(Arc<AtomicBool>);
 
 impl Builder {
-    pub fn new(addr: SocketAddr, f: WorkloadInformation) -> Self {
+    pub fn new(f: WorkloadInformation) -> Self {
```

Comment:
Maybe need to pass address later

Reply:
Yeah we should put this in Config. Even though it's not configurable right now, in the future we will make everything in Config customizable

Reply:
Okay, got it. I handled the conflict and re-submitted.